doc/dev/installation.rst

```diff
@@ -414,3 +414,123 @@ To generate html files for the documentation for Catalyst:
+  other virtual environments:
+
+  .. code-block:: console
+
+    python3 -m venv /tmp/myenv
+    source /tmp/myenv/bin/activate
+
+- As a clean-up measure, Uninstall any previous Catalyst instances of that particular virtual
+  environment:
+
+  .. code-block:: console
+
+    python -m pip uninstall -y pennylane-catalyst
+
+- Install the latest CPL suite wheels from the TestPyPI server:
+
+  .. code-block:: console
+
+    python -m pip install --extra-index-url https://test.pypi.org/simple/ pennylane pennylane-lightning pennylane-catalyst --pre --upgrade
+
+- Search in the Git repository for the latest commit whose version corresponds to the 
+  Catalyst wheel version. This will leave the repository at a "detached head" state.
+
+- Once the Catalyst wheel and the repository are at the same version, create hard links from 
+  the Catalyst wheel code to the frontend code of the repository, in order to allow working
+  directly with the frontend repository codebase and at the same time test the changes while
+  using the installed Catalyst wheel libraries, hence avoiding compilation:
+
+  .. code-block:: console
+
+    export SITEPKGS=$(python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
+    export CATALYST_WHEEL=$SITEPKGS/catalyst
+    cp -lrf $CATALYST_WHEEL .frontend/
+
+Further Steps
+^^^^^^^^^^^^^
+
+If everything goes well, ``git status`` should not report any changed files. 
+
+Before making changes to the frontend, make sure you create a new branch:
+
+.. code-block:: console
+
+  git checkout -b new-branch-name
+
+Once in the new branch, make the wanted changes. Use the IDE of your preference.
+
+You can test the changes by executing your sample code under the same virtual environment you used
+with the scripts. As you are actually directly changing the code stored at the Python ``site-packages``
+folder, you will be automatically using the shared libraries provided by the Python wheels. Again,
+there is no need to compile Catalyst from source.
+
+When you are ready, commit your changes as usual. As your branch was created from a detached head,
+you can choose to rebase your branch according to the ``main`` branch:
+
+.. code-block:: console
+  
+  git fetch
+  git rebase -i origin/main
+
+Resolve any conflicts and once ready, push the new branch to the remote repository:
+
+.. code-block:: console
+  
+  git push -u origin new-branch-name
+
+Now you can go to GitHub and issue a Pull Request based on the new branch.
+
+Special Considerations
+^^^^^^^^^^^^^^^^^^^^^^
+
+Due to the fact that the scripts are part of the Git repository, when going backwards in the commit history
+the scripts might have also changed to a previous version. Therefore, if you want to run the scripts again,
+first, make sure you have commited your changes to a branch (as suggested in the previous section), and 
+second, return back to the latest state of the main branch:
+
+.. code-block:: console
+  
+  git switch main
+  git fetch
+  git pull
```

Comment:
I don't quite understand the intention of this section, is it just do describe how to run the latest version of the dev_wheel script ?

Reply:
Yes, but I guess it is not a big deal at the moment. Removed. Done.